hack/builder/Dockerfile

```diff
@@ -130,10 +141,13 @@ COPY entrypoint.sh /entrypoint.sh
 
 COPY create_bazel_cache_rcs.sh /create_bazel_cache_rcs.sh
 
-RUN curl -L -o /usr/bin/bazel https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-${BAZEL_ARCH} && \
-    chmod u+x /usr/bin/bazel
+RUN if test "${ARCH}" != "s390x"; then \
+       curl -L -o /usr/bin/bazel https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-${BAZEL_ARCH} && \
+    chmod u+x /usr/bin/bazel; \
+    fi
 
 # Add /root/go/src/kubevirt.io/kubevirt and mark it as a safe directory for git
 RUN mkdir -p /root/go/src/kubevirt.io/kubevirt && git config --global --add safe.directory /root/go/src/kubevirt.io/kubevirt
 
 ENTRYPOINT [ "/entrypoint.sh" ]
+
```

Comment:
nit - no need for this new line here

Reply:
I think adding newlines at the end of a file if there is none is the default setting on most modern editors.
So they might not be required, but are mostly enforced by editors by default...

Reply:
removed the new line.